test/integration_test.rb

```diff
@@ -0,0 +1,62 @@
+require 'helpers/kubeclient_helper'
+
+module KubernetesDeploy
+  class IntegrationTest < ::Minitest::Test
+    include KubeclientHelper
+
+    def setup
+      @logger_stream = StringIO.new
+      @logger = Logger.new(@logger_stream)
+      KubernetesDeploy.logger = @logger
```

Comment:
I still got a lot of logs to `stdout/stderr` when running the tests locally:

```
[KUBESTATUS] {"group":"Web deployment","name":"web-3766115818-x7ns1","status_string":"Pending (Ready: false)","exists":true,"succeeded":false,"failed":false,"timed_out":false}
```

And so on with the test output